src/main/drivers/serial_uart_pinconfig.c

```diff
@@ -53,14 +53,31 @@ void uartPinConfigure(const serialPinConfig_t *pSerialPinConfig)
         const uartHardware_t *hardware = &uartHardware[hindex];
         const UARTDevice_e device = hardware->device;
 
+#if !(defined(STM32F1) || defined(STM32F4)) // Older CPUs don't support pin swap.
```

Comment:
Seeing that `pinSwap` is never initialised to false for all other MCU types, this should either be done unconditionally, or omitted.

Reply:
`uartDevice_t`, the structure that contains `pinSwap`, only contains it if the processor supports pin swap, so no initialisation to do.  I think the call to `uartConfigurePinSwap` needs gating thought, I'll fix that.

Reply:
Done